web/src/gql/provider.tsx

```diff
@@ -37,6 +38,7 @@ const Provider: React.FC<Props> = ({ children }) => {
     if (!networkError && !graphQLErrors) return;
     const error = networkError?.message ?? graphQLErrors?.map(e => e.message).join(", ");
     if (error) {
+      Notification.error({ message: error });
```

Comment:
Since you are using Notification.error in places of errors, I think you can remove the setError below this and remove that completely since I dont think we are using that then.

Reply:
https://github.com/reearth/reearth-cms/blob/d5369fdbd6bba4c3caf029b5daf5e77166d37ac5/web/src/gql/provider.tsx#L38-L42
Removed it, with all it's occurrences in the `state` file